pkg/agent/plugin/workloadattestor/k8s/k8s_posix.go

```diff
@@ -69,13 +85,17 @@ func (h *containerHelper) GetOSSelectors(ctx context.Context, log hclog.Logger,
 	return selectors, nil
 }
 
-func (h *containerHelper) GetPodUIDAndContainerID(pID int32, _ hclog.Logger) (types.UID, string, error) {
-	cgroups, err := cgroups.GetCgroups(pID, h.fs)
-	if err != nil {
-		return "", "", status.Errorf(codes.Internal, "unable to obtain cgroups: %v", err)
+func (h *containerHelper) GetPodUIDAndContainerID(pID int32, log hclog.Logger) (types.UID, string, error) {
+	if !h.useNewContainerLocator {
+		cgroups, err := cgroups.GetCgroups(pID, dirFS(h.rootDir))
```

Comment:
I thought about it, but it's a lot of tests to duplicate. I thought maybe since this is deprecated and not going to change, that it might be ok..... what do you think?

Reply:
I'm not sure, we still have IT that is using this, so we may be fine with that
and we can just add another IT in the future to know it is till working until we remove?

Reply:
I've updated the IT tests to use the new locator.